--- origsrc/webrtc-audio-processing-0.3/webrtc/modules/audio_processing/Makefile.am	2016-06-21 06:15:34.000000000 -0500
+++ src/webrtc-audio-processing-0.3/webrtc/modules/audio_processing/Makefile.am	2016-07-27 21:13:47.154803700 -0500
@@ -175,7 +175,7 @@ libwebrtc_audio_processing_la_LIBADD = $
 				       $(LIBWEBRTC_PRIVATEARCH) \
 				       -lpthread
 libwebrtc_audio_processing_la_LDFLAGS = $(AM_LDFLAGS) \
-					-Wl,--no-undefined \
+					-no-undefined \
 					-version-info $(LIBWEBRTC_AUDIO_PROCESSING_VERSION_INFO)
 
 # FIXME: The MIPS optimisations need to be hooked up once we have the
